wp-content/themes/core/components/blocks/card_grid/css/card_grid.pcss

```diff
@@ -0,0 +1,38 @@
+/* -----------------------------------------------------------------------------
+ *
+ * Component: Block: Card Grid
+ *
+ * ----------------------------------------------------------------------------- */
+
+.b-card-grid {}
+
+/* -----------------------------------------------------------------------------
+ * Block Header
+ * ----------------------------------------------------------------------------- */
+
+.b-card-grid__header {
+	/* CASE: Layout Stacked */
+	.b-card-grid--layout-stacked & {
+		margin-bottom: var(--spacer-60);
+	}
+}
+
+/* -----------------------------------------------------------------------------
+ * Cards Loop
+ * ----------------------------------------------------------------------------- */
+
+.b-card-grid__loop {
+	display: grid;
+	grid-gap: var(--grid-gutter-small);
+	grid-template-columns: repeat(auto-fit, minmax(min(var(--grid-3-col), 100%), var(--grid-4-col)));
+	/*grid-template-columns: repeat(auto-fill, minmax(var(--grid-3-col), var(--grid-4-col)));*/
```

Comment:
Is this needed? Perhaps a doc note if so?

Reply:
Good catch. It is not needed at all. Removed.